packaging/hudi-integ-test-bundle/pom.xml

```diff
@@ -482,6 +482,12 @@
       <artifactId>hadoop-hdfs</artifactId>
       <classifier>tests</classifier>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.eclipse.jetty</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Okay, so a general high level comment here. Throughout the PR there are jetty dependency exclusions added which is creating a lot of unnecessary diff. Ideally this should all just be managed through the root POMs `dependencyManagement` instead of excluding it everywhere. We should clean up these diffs from the PR.